pages/index.js

```diff
@@ -7,6 +8,9 @@ export default function HomePage({
 }) {
   return (
     <section>
+      <Link href="/createForm">
+        <button type="button"> + New</button>
+      </Link>
       <ActivitiesList
         activities={activities}
```

Comment:
I think 'activities' is not needed since all information is included in 'activitiesInfo'.